site/index.js

```diff
@@ -37,3 +39,21 @@ function checkForTab(e) {
     md.selectionEnd = start + 1;
   }
 }
+
+function read_md_file(file)
+{
+    var rawFile = new XMLHttpRequest();
```

Comment:
In the functions above variables are declared with let. I would suggest to use  let in situations where it doesn't matter if you use var or let.

Reply:
okay good point